STROOP/Utilities/Stream/WindowsProcessIO.cs

```diff
@@ -90,9 +91,19 @@ bool CompareBytes(byte[] a, byte[] b)
                     return false;
             return true;
         }
-
+        
         protected virtual void CalculateOffset()
         {
+            // Find CORE_RDRAM export from mupen if present
+            Win32SymbolInfo smybolInfo = Win32SymbolInfo.Create();
+            if (SymInitialize(_process.Handle, null, true) && SymFromName(_process.Handle, "CORE_RDRAM", ref smybolInfo))
+            {
+                var val = new byte[4];
```

Comment:
This won't work for x64 builds of Mupen64; the pointer would have width 8, not 4.